inference-engine/src/transformations/src/transformations/op_conversions/log_softmax_decomposition.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/op_conversions/log_softmax_decomposition.hpp"
+
+#include <memory>
+
+#include <ngraph/opsets/opset5.hpp>
+#include <ngraph/rt_info.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::LogSoftmaxDecomposition, "LogSoftmaxDecomposition", 0);
+
+ngraph::pass::LogSoftmaxDecomposition::LogSoftmaxDecomposition() {
+    // Decomposes LogSoftmax(x, axis) op into sub-graph x - log(reduce_sum(exp(x), axis))
+    auto log_softmax = ngraph::pattern::wrap_type<opset5::LogSoftmax>();
```

Comment:
Do we have tests for inference results check against this transformation?

Reply:
We have transformation tests, but I am not aware of computational tests. Never the less the correctness of this decomposition was proved before since previously MO was converting LogSoftmax to the same set of operations.